test/extensions/wasm/wasm_test.cc

```diff
@@ -175,9 +175,9 @@ TEST_P(WasmTest, EmscriptenVersion) {
   uint32_t major = 9, minor = 9, abi_major = 9, abi_minor = 9;
   EXPECT_TRUE(wasm->getEmscriptenVersion(&major, &minor, &abi_major, &abi_minor));
   EXPECT_EQ(major, 0);
-  EXPECT_EQ(minor, 0);
+  EXPECT_TRUE(minor >= 0 && minor <= 1);
   EXPECT_EQ(abi_major, 0);
-  EXPECT_EQ(abi_minor, 1);
+  EXPECT_TRUE(abi_minor >= 0 && abi_minor <= 3);
```

Comment:
Unrelated changes?

Reply:
The provides support for up to 1.38.39 of emscripten.  Yes, it is unrelated.  I can pull it out if you want.

Reply:
I'd prefer to have separate PRs for working features that clearly describe intention behind the change, than sneaked in changes and/or code dumps.

Reply:
OK, I'll pull that out.

Reply:
https://github.com/envoyproxy/envoy-wasm/pull/83